src/main/kotlin/org/llamarama/petty/blocks/ModIdBlocks.kt

```diff
@@ -35,6 +37,12 @@ object ModIdBlocks {
         return block
     }
 
+    private fun <B: Block> addSpecialItemBlock(name: String, block: B): B {
```

Comment:
The generic is actually not needed because of inheritance, but I forgot. The current version of the master branch removed the generic and just has block: Block.
I suggest we do the same here.

Reply:
Yeah let's remove it - I was just working from an older version and forgot to fetch the changes that you made.